src/services/web-monitoring-db.js

```diff
@@ -55,12 +55,27 @@ const storageLocation = 'WebMonitoringDb.token';
  */
 
 /**
- * @typedef ApiResponse
+ * @typedef {Object} ApiResponse
```

Comment:
FWIW, the `{type}` part of a `@typedef` [is supposed to be optional](http://usejsdoc.org/tags-typedef.html). It’s totally fine to add it, though — no need to change this back.

(The main reason to use it is to suggest that a type *isn’t* just an object, but a particular kind of `string` or `number` or other specific type. e.g. `@typedef {number} Milliseconds` so you can `@param {Milliseconds} delay` when defining a function.)

Reply:
I added it to be consistent with how all the other objects were defined. I think it makes a lot more sense to add a type for the reason you described. `{Object}` is so generic, it doesn't provide any useful information. 

Seems like we agree here, so I removed `{Object}` from all the other type definitions.